code/index.html

```diff
@@ -1,30 +1,135 @@
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/e51864a347.js" crossorigin="anonymous"></script>
+    <title>Technigo P02: Ziko's Org.</title>
+</head>
+
+<body>
+
+    <main id="main-content">
+
+        <section class="hero">
+            <article id="splash">
+                <h1>
+                    Make a choice.
+                </h1>
+                <p>
+                    To perish into smoldering ash, or to persevere on the brink of ruin.
+                </p>
+                <a href="#sign-up" alt="Click here to sign up"><button class="btn" id="sign-up-btn">Persevere!</button></a>
+            </article>
+        </section>
+
+        <section class="bridge">
+            <article id="ingress">
+                <p>
+                    The best time to plant a tree was twenty years ago, the second best time is today.
+                </p>
+            </article>
+        </section>
+
+        <section class="boxes">
+            <article id="sign-up">
+                <h1>
+                    Sign up - we just need a few deets:
+                </h1>
+                <form action="https://httpbin.org/anything" method="POST">
+                    <ul>
+                        <li>
+                            <input type="text" name="user-name" placeholder="What you go by" required>
+                        </li>
+                        <li>
+                            <input type="email" name="user-email" placeholder="E-communique" required>
+                        </li>
+                        <li><p>And what are you?</p>
+                            <input type="radio" name="being" id="human-bean" value="Human bean" required>
+                            <label for="human-bean">
+                                Human bean
+                            </label><br>
+                            <input type="radio" name="being" id="ethereal-spirit" value="Ethereal spirit" required>
+                            <label for="ethereal-spirit">
+                                Ethereal spirit
+                            </label><br>
+                            <input type="radio" name="being" id="doggo" value="Doggo" required>
+                            <label for="doggo">
+                                Doggo
+                            </label><br>
+                            <input type="radio" name="being" id="mycelium" valie="Mycelium" required>
```

Comment:
A typo - valie. 
This made the radio button not work.

Reply:
Haha, the more eyes the better! Thank you for pointing this out 👍